scripts/create-package.sh

```diff
@@ -0,0 +1,243 @@
+    case $platform in
+        linux|msys2) install -DT -m 644 "$src" "$dest" ;;
+        msvc) create_parent_dir "$dest" && cp "$src" "$dest" ;;
+        macos) create_parent_dir "$dest" && install -m 644 "$src" "$dest" ;;
+    esac
+}
+
+install_doc()
+{
+    # Install common documentation files
+    case $platform in
+        linux)
+            install_file docs/README.template "${pkg_dir}/README"
+            install_file COPYING              "${pkg_dir}/COPYING"
+            install_file README               "${pkg_dir}/doc/manual.txt"
+            install_file docs/dosbox.1        "${pkg_dir}/man/dosbox.1"
+            readme_tmpl="${pkg_dir}/README"
+            ;;
+        macos)
+            install_file docs/README.template "${macos_dst_dir}/SharedSupport/README"
+            install_file COPYING              "${macos_dst_dir}/SharedSupport/COPYING"
+            install_file README               "${macos_dst_dir}/SharedSupport/manual.txt"
+            install_file docs/README.video    "${macos_dst_dir}/SharedSupport/video.txt"
+            readme_tmpl="${macos_dst_dir}/SharedSupport/README"
+            ;;
+        msys2|msvc)
+            install_file COPYING              "${pkg_dir}/COPYING.txt"
+            install_file docs/README.template "${pkg_dir}/README.txt"
+            install_file docs/README.video    "${pkg_dir}/doc/video.txt"
+            install_file README               "${pkg_dir}/doc/manual.txt"
+            readme_tmpl="${pkg_dir}/README.txt"
+            ;;
+    esac
+    # Fill template variables in README.template
+    [ -n "$git_commit" ] && sed -i -e "s|%GIT_COMMIT%|$git_commit|" "$readme_tmpl"
+    [ -n "$git_branch" ] && sed -i -e "s|%GIT_BRANCH%|$git_branch|" "$readme_tmpl"
+    [ -n "$git_repo" ]   && sed -i -e "s|%GITHUB_REPO%|$git_repo|"  "$readme_tmpl"
+}
+
+install_translation()
+{
+    lng_dir=${pkg_dir}/translations
+    if [ "$platform" = "macos" ]; then
+        lng_dir=${macos_dst_dir}/Resources/translations
+    fi
+    # Prepare translation files
+    #
+    # Note:
+    #   We conciously drop the dialect postfix because no dialects are available.
+    #   (US was the default DOS dialect and therefore is the default for 'en').
+    #   There users get the generic translation and benefit from simpler filenames.
+    #   Dialect translations will be added if/when they're available.
+    #
+    install_file contrib/translations/de/de_DE.lng       "$lng_dir/de.lng"
+    install_file contrib/translations/en/en_US.lng       "$lng_dir/en.lng"
+    install_file contrib/translations/es/es_ES.lng       "$lng_dir/es.lng"
+    install_file contrib/translations/fr/fr_FR.lng       "$lng_dir/fr.lng"
+    install_file contrib/translations/it/it_IT.lng       "$lng_dir/it.lng"
+    install_file contrib/translations/pl/pl_PL.CP437.lng "$lng_dir/pl.cp437.lng"
+    install_file contrib/translations/pl/pl_PL.lng       "$lng_dir/pl.lng"
+    install_file contrib/translations/ru/ru_RU.lng       "$lng_dir/ru.lng"
+}
```

Comment:
If this could be made dynamic, then it's making added/removing translations even fewer steps (or places where we accidentally missing something):

``` bash
for source in $(find contrib/translations -name '*.lng'); do
    target=$(basename "$source" | tr '[:upper:]' '[:lower:]' | sed 's/_[a-z][a-z]//')
    install_file "$source" "$lng_dir/$target"
done
```

Hopefully that works on macOS too.

Reply:
Good idea, I made a modification to the loop to remove the `sed` requirement. Hopefully that works for you.